code/datums/gamemode/gamemode.dm

```diff
@@ -70,7 +72,7 @@
 	return dat
 
 /datum/gamemode/proc/TearDown()
-	//No idea what this is supposed to do
+	//No idea what this is supposed to do - it probably the datumised version of "error setting up secret." -- t. Shifty
```

Comment:
This is where the game mode is shut down and cleaned up.  SetUp vs. TearDown are commonly used in unit testing parlance.